satellite/console/service.go

```diff
@@ -441,39 +460,38 @@ func (s *Service) CreateProject(ctx context.Context, projectInfo ProjectInfo) (p
 		return
 	}
 
-	project := &Project{
-		Description: projectInfo.Description,
-		Name:        projectInfo.Name,
-	}
-
-	transaction, err := s.store.BeginTx(ctx)
+	tx, err := s.store.BeginTx(ctx)
 	if err != nil {
 		return nil, errs.New(internalErrMsg)
 	}
 
+	// clean up project on error
 	defer func() {
```

Comment:
same about this defer. if someone changes the named return value names, everything will still compile, but this defer method will stop working right. instead, if this was at the top of the function, then it wouldn't compile if someone changed the names.